pkgs/by-name/po/porn-vault/package.nix

```diff
@@ -0,0 +1,103 @@
+{
+  fetchFromGitLab,
+  fetchurl,
+  rustPlatform,
+  lib,
+  pnpm,
+  stdenvNoCC,
+  nodejs_22,
+  ffmpeg,
+  imagemagick,
+  writeShellScript,
+  makeWrapper,
+  autoPatchelfHook,
+}:
+let
+  izzy = rustPlatform.buildRustPackage rec {
```

Comment:
I think the amount of code that goes in here justifies moving izzy to its own package x3

Reply:
Nah. Izzy is made by the porn-vault dev and only used by porn-vault. Additionally, izzy is archived and will be deprecated on the next Porn-Vault version.
Having it on a separate derivation would not only not make sense, but would also make it a pain in the ass to remove it in just a few months when it’s ditched. :c